gx_suite_definitions/biomarkers.ipynb

```diff
@@ -0,0 +1,239 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "biomarkers_data_file = syn.get(\"syn63540269\").path"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create Validator Object on Data File"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.read_json(biomarkers_data_file)\n",
+    "nested_columns = ['points']\n",
+    "df = GreatExpectationsRunner.convert_nested_columns_to_json(df, nested_columns)\n",
+    "validator = context.sources.pandas_default.read_dataframe(df)\n",
+    "validator.expectation_suite_name = \"biomarkers\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Add Expectations to Validator Object For Each Column"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# points\n",
+    "validator.expect_column_values_to_be_of_type(\"points\", \"str\")\n",
+    "with open(\"../src/agoradatatools/great_expectations/gx/json_schemas/immunohisto/points.json\", \"r\") as file:\n",
+    "    points_schema = json.load(file)\n",
+    "validator.expect_column_values_to_match_json_schema(\"points\", json_schema=points_schema)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# model\n",
+    "validator.expect_column_values_to_be_of_type(\"model\", \"str\")\n",
+    "validator.expect_column_values_to_not_be_null(\"model\")\n",
+    "# allows all alphanumeric characters, underscores, periods, and dashes\n",
+    "validator.expect_column_values_to_match_regex(\"type\", \"^[A-Za-z0-9\\s\\*_.-]+$\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# type\n",
+    "validator.expect_column_values_to_be_of_type(\"type\", \"str\")\n",
+    "validator.expect_column_values_to_not_be_null(\"type\")\n",
+    "# allows all alphanumeric characters, underscores, periods, and dashes\n",
+    "validator.expect_column_values_to_match_regex(\"type\", \"^[A-Za-z0-9\\s_.-]+$\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# units\n",
+    "validator.expect_column_values_to_be_of_type(\"units\", \"str\")\n",
+    "validator.expect_column_values_to_not_be_null(\"units\")\n",
+    "# allows all alphanumeric characters, underscores, periods, and dashes\n",
+    "validator.expect_column_values_to_match_regex(\"units\", \"^[A-Za-z0-9\\/\\s\\*_.-]+$\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# age_death\n",
+    "validator.expect_column_values_to_be_of_type(\"age_death\", \"int\")\n",
```

Comment:
Question for @JessterB : Will age_death always be an integer value? I've noticed in the metadata files that some people round and some people use exact values (i.e. 11.33 mo), not sure if this is the case for pathology/biomarkers.

Reply:
I think I've only seen integer values in the source data. If they give us exact values in the future, we'd want to round them for display anyway, so we would probably do that when we incorporate that additional data into the harmonized source file.

Reply:
Sounds like we are keeping it as an integer. Let me know if I misunderstood!

Reply:
Yes, keep integer!